ccmlib/node.py

```diff
@@ -1427,6 +1427,16 @@ def __update_envfile(self):
         if self.get_cassandra_version() < '2.0.1':
             common.replace_in_file(conf_file, "-Xss", '    JVM_OPTS="$JVM_OPTS -Xss228k"')
 
+        # gc.log was turned on by default in 2.2.5/3.0.3/3.3
+        if self.get_cassandra_version() >= '2.2.5':
+            gc_log_pattern = "-Xloggc"
+            if common.is_win():
+                gc_log_setting = '    $env:JVM_OPTS="$env:JVM_OPTS -Xloggc:$env:{}/logs/gc.log"'.format(self.get_path())
+            else:
+                gc_log_setting = 'JVM_OPTS="$JVM_OPTS -Xloggc:{}/logs/gc.log"'.format(self.get_path())
```

Comment:
Since the `' -Xloggc:{}/logs/gc.log"'.format(self.get_path())'` part of both of these strings is identical, part of me wants to ask you to pull that out so you only write the `-Xloggc` option part once and there's less danger of incorrectly changing one but not the other in the future. That may be a silly request though; I trust your judgement.